### PR TITLE
Added update titles function

### PR DESCRIPTION
**Pull request by simakover**
Added update titles fuction, logic is same as update cover. 
Im have an issue a long ago with my manga list, there are many mangas have wrong titles. Im dont know deserve this changes pull or not, but im made it) 
Also this is my first try in andriod dev\forking, so any advices welcome

### CODE REVIEW

**Comment by paulo27ms** [comment]
Is updating titles generally a good idea? Does this break download folders?

File: app/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -529,6 +529,7 @@
     </plurals>
     <string name="battery_optimization_setting_activity_not_found">Не удалось открыть настройки устройства</string>
     <string name="pref_refresh_library_covers">Обновить обложки библиотечной манги</string>
+    <string name="pref_refresh_library_titles">Обновить заголовки библиотечной манги</string>
```
Comment (Riztard) [inline app/src/main/res/values-ru/strings.xml:532]: i believe for other lang strings, you can only change it through weblate
or it will break something

**Comment by Riztard** [comment]
why not just use refresh metadata when update library, did it not work? 🤔

**Comment by simakover** [comment]
> why not just use refresh metadata when update library, did it not work? 🤔

not working for titles

File: app/src/main/res/values-ru/strings.xml (hunk 1 above)
Comment (simakover) [reply app/src/main/res/values-ru/strings.xml]: revert changes in commit 009dc4539c4f6f5f1eb927f04b67146b9b2ee486

**Comment by simakover** [comment]
Looks like after changing app/src/main/java/eu/kanade/tachiyomi/source/model/SManga.kt with adding title it start updating via metadata, but sometimes im have error: "lateinit property title has not been initialized"

**Pull request closed without merging**